README.md

```diff
@@ -27,7 +27,9 @@ Please refer to the <a href="https://docs.espressif.com/projects/espressif-ide/e
 	```
 	git clone https://github.com/espressif/idf-eclipse-plugin.git
 	cd idf-eclipse-plugin
-	mvn clean verify -Djarsigner.skip=true
+	mvn clean verify -Djarsigner.skip=true \
+	  -Djdk.xml.maxGeneralEntitySizeLimit=0 \
+	  -Djdk.xml.totalEntitySizeLimit=0
```

Comment:
Thanks for the PR. While this can help bypass errors with large XML files, setting these limits to zero disables important protections against XML. These limits are in place to prevent malicious or accidental resource exhaustion 

Best practice is to only increase these limits as much as necessary when needed, not to remove them entirely. If we are hitting the default limits, we should set them to a large but finite value that is sufficient for our use case (e.g., -Djdk.xml.totalEntitySizeLimit=200000000 for 200MB), rather than disabling the limit.

For now, I am not merging this change. 

Thanks again for your effort and understanding!